presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -41,14 +43,16 @@
 import static io.prestosql.plugin.hive.HiveErrorCode.HIVE_BAD_DATA;
 import static io.prestosql.plugin.hive.HiveErrorCode.HIVE_CURSOR_ERROR;
 import static io.prestosql.spi.type.BigintType.BIGINT;
-import static io.prestosql.spi.type.IntegerType.INTEGER;
-import static java.lang.Math.toIntExact;
 import static java.util.Objects.requireNonNull;
 import static org.apache.hadoop.hive.ql.io.AcidUtils.deleteDeltaSubdir;
 
 @NotThreadSafe
 public class OrcDeletedRows
 {
+    private static final int ORIGINAL_TRANSACTION_INDEX = 0;
+    private static final int ROW_ID_INDEX_DELETE_DELTA = 1; // delete delta page has columns: <originalId,rowId>
+    private static final int ROW_ID_INDEX = 2; // base/delta page has columns: <originalId,bucketId,rowId>
```

Comment:
If we are do not care about bucket id from base/delta pages can't we just refrain from setting up PageSource from reading it. 

I.e. this 
```
                fileReadColumns.add(acidColumnsByName.get(ACID_COLUMN_BUCKET.toLowerCase(ENGLISH)));
                fileReadTypes.add(INTEGER);
                fileReadLayouts.add(fullyProjectedLayout());
```
from https://github.com/prestosql/presto/blob/c0ead8f19be0930a5ad820033de77a34dd9fe125/presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcPageSourceFactory.java#L192 ?

Then we could use same index. Right?

Reply:
yeah, I have pushed the changes.